recipe/run_test.sh

```diff
@@ -0,0 +1,8 @@
+# Skip compile tests if we are cross-compiling
+if [[ "${build_platform}" != "${target_platform}" ]]; then
+  exit 0
+fi
+
+# Test boost::iostreams zlib filter support
+cd test
+${CXX} -I$PREFIX/include -L$PREFIX/lib test_iostreams_zlib.cpp -lboost_iostreams && ./a.out
```

Comment:
Maybe we need to set the `rpath`? Mentioning since the macOS build fails with a library loading error.

```suggestion
${CXX} -I$PREFIX/include -L$PREFIX/lib -Wl,-rpath,$PREFIX/lib test_iostreams_zlib.cpp -lboost_iostreams && ./a.out
```

Reply:
Yes, I was just looking at that too. Your suggestion seems reasonable to me - let's try it.